src/shared/pam-util.c

```diff
@@ -51,19 +51,41 @@ int pam_syslog_pam_error(pam_handle_t *handle, int level, int error, const char
         return error;
 }
 
-static void cleanup_system_bus(pam_handle_t *handle, void *data, int error_status) {
-        /* The PAM_DATA_SILENT flag is the way that pam_end() communicates to the module stack that this
-         * invocation of pam_end() is not the final one, but in the process that is going to directly exec
-         * the child. This means we are being called after a fork(), and we do not want to try and clean
-         * up the sd-bus object, as it would affect the parent too and we'll hit an assertion. */
-        if (error_status & PAM_DATA_SILENT)
```

Comment:
removing this check means we won't detect leaks anymore

Reply:
"leaks"? i mean we are not in the business of that, are we?

Reply:
I mean places where we should have done the cleanup, but forgot - like the homed one I fixed last week. IOW, if this is called, it means there is a programming error somewhere

Reply:
kept it in as a warning now